structuralcodes/geometry/_circular.py

```diff
@@ -0,0 +1,173 @@
+"""Classes for circular geometries.
+
+The class `CircularGeometry` represents a circular SurfaceGeometry with
+homogenous material.
+This class is simply a wrapper of `SurfaceGeometry` class and permits an easy
+input by the user.
+"""
+
+import typing as t
+from itertools import chain
+
+import numpy as np
+from shapely import Polygon
+
+from structuralcodes.core.base import ConstitutiveLaw, Material
+
+from ._geometry import CompoundGeometry, SurfaceGeometry
+from ._reinforcement import add_reinforcement_circle
+
+
+def _create_circle(radius, npoints=20):
+    """Create a circle with a given radius."""
+    phi = np.linspace(0, 2 * np.pi, npoints + 1)
+    x = radius * np.cos(phi)
+    y = radius * np.sin(phi)
+    points = [(this_x, this_y) for this_x, this_y in zip(x, y)]
+    return Polygon(points)
+
+
+class CircularGeometry(SurfaceGeometry):
+    """This is a wrapper class for defining a SurfaceGeometry of circular shape
+    with a homogeneous material.
+    """
+
+    _radius: float
+
+    def __init__(
+        self,
+        size: float,
+        material: t.Union[Material, ConstitutiveLaw],
+        is_radius: bool = False,
+        n_points: int = 20,
+        density: t.Optional[float] = None,
+        concrete: bool = False,
+    ) -> None:
+        """Initialize a CircularGeometry.
+
+        Arguments:
+            size (float): The size of the geometry. By default this number
```

Comment:
Can we do either radius or diameter and use either `radius` or `diameter` as argument? Saves an argument, and makes it explicit for the user 👍 Since we are already using diameter when adding reinforcement, maybe we can use that also here?

Reply:
I am not sure I got your proposal. You are suggesting to accept only `diameter` for instance and avoid this mechanism for accepting diameter or radius? Totally fine for me. I can do it for sure!

Reply:
Exactly that 😃 At least to me, having just diameter, and skipping the `is_radius`, makes i clearer what to insert.

Reply:
Sure. `is_radius` was optional anyhow so only if you wanted to insert the radius you would have need to worry about setting it. Anyhow I agree that the api is simpler when selecting just one. And the user can always do its division. I implemented only `diameter` in f635cb3b6e4c9f23fffbc0f3f034c32667f4db51. Anyhow I maintained both radius and diameter properties if those could be of any use.